lib/src/parse/stylesheet.dart

```diff
@@ -1729,16 +1752,22 @@ abstract class StylesheetParser extends Parser {
   /// If [until] is passed, it's called each time the expression could end and
   /// still be a valid expression. When it returns `true`, this returns the
   /// expression.
+  ///
+  /// If [allowNewlines] is true, the indented syntax will consume newlines as
+  /// whitespace, in positions when a statement can not end.
```

Comment:
This should be more explicit that it only refers to the top level of the expression, since expressions can have bracketed sub-expressions that always allow newlines.